### PR TITLE
Enable config

### PR DESCRIPTION
**Pull request by gazpachoking**
Simple implementation of `/config` endpoint.

PUT overwrites whole config with provided config. (so you can delete keys)
PATCH just updates given keys.

### CODE REVIEW

File: spritzle/resource/config.py [hunk 1]
```diff
@@ -20,10 +20,28 @@
...
+@routes.put('/config')
+async def put_config(request):
+    config = request.app['spritzle.config']
+    new_values = await request.json()
+    config.data = new_values
+    return web.Response()
 
 
+@routes.patch('/config')
 async def put_config(request):
```
Comment (aresch) [inline spritzle/resource/config.py]: This should be patch_config(request)

File: spritzle/resource/config.py [hunk 2]
```diff
@@ -20,10 +20,28 @@
...
-    raise NotImplementedError
+    config = request.app['spritzle.config']
+    return web.json_response(dict(config))
+
+
+@routes.put('/config')
+async def put_config(request):
+    config = request.app['spritzle.config']
+    new_values = await request.json()
+    config.data = new_values
```
Comment (aresch) [inline spritzle/resource/config.py]: I think you should call config.save() after setting the data.

**Review by aresch — CHANGES_REQUESTED** [review]

File: spritzle/resource/config.py (hunk 1 above)
Comment (gazpachoking) [reply spritzle/resource/config.py]: Oops. Silly copy/paste error.

File: spritzle/resource/config.py (hunk 2 above)
Comment (gazpachoking) [reply spritzle/resource/config.py]: Good call.

**Review by aresch — APPROVED** [review]

**Pull request merged**